torchaudio/functional.py

```diff
@@ -558,37 +557,37 @@ def lfilter(waveform, a_coeffs, b_coeffs):
 
     device = waveform.device
     dtype = waveform.dtype
-    n_channels, n_frames = waveform.size()
+    n_channel, n_sample = waveform.size()
```

Comment:
For consistency, `channel` and `frame`, rather than `n_channel` and `n_sample`?

Reply:
We use `n_*` to refer to the actual value, and `*` to the name of the dimension. Based on this convention, this would be ok. We could decide to change the convention itself, but I'd leave that to a separate PR if at all.